pkg/chargeback/http.go

```diff
@@ -448,16 +460,15 @@ type ReportResultValues struct {
 }
 
 func convertsToGetReportResults(input []map[string]interface{}, columns []api.ReportGenerationQueryColumn) GetReportResults {
+	// converts to json/csv report with more fieldnames and ability to modify results if tableHidden is false
```

Comment:
I would rephrase this as it's not super clear atm, and secondly, in Go you generally put the comment above the function name, and start with the function name's name.

```
// convertsToGetReportResults converts maps returned from `presto.ExecuteSelect` into a GetReportResults.
```